eng/Versions.props

```diff
@@ -86,10 +86,10 @@
   </PropertyGroup>
   <!-- XUnit-related (not extensions) -->
   <PropertyGroup>
-    <XUnitVersion>2.4.0</XUnitVersion>
```

Comment:
I updated the nugets to fix some version conflicts (I used the same version as WinForms).